tensornetwork/ncon_interface_test.py

```diff
@@ -148,12 +290,26 @@ def test_contraction(backend):
   np.testing.assert_allclose(res, res_np)
 
 
-def test_backend_network(backend):
-  a = np.random.randn(2, 2, 2)
-  tn, _, _ = ncon_interface.ncon_network([a, a, a], [(-1, 1, 2), (1, 2, 3),
-                                                     (3, -2, -3)],
-                                         backend=backend)
-  res = naive(tn).get_final_node().get_tensor()
-  res_np = a.reshape((2, 4)) @ a.reshape((4, 2)) @ a.reshape((2, 4))
+def test_node_contraction(backend):
+  tensor = np.random.randn(2, 2, 2)
+  a = Node(tensor, backend=backend)
+  res = ncon_interface.ncon([a, a, a], [(-1, 1, 2), (1, 2, 3), (3, -2, -3)],
+                            backend=backend)
+  res_np = tensor.reshape((2, 4)) @ tensor.reshape((4, 2)) @ tensor.reshape(
+      (2, 4))
   res_np = res_np.reshape((2, 2, 2))
-  np.testing.assert_allclose(res, res_np)
+  np.testing.assert_allclose(res.tensor, res_np)
+
+
+# def test_backend_network(backend):
```

Comment:
Delete commented out code

(Why was this deleted)?

Reply:
added it back